### PR TITLE
Fix footer links

### PR DESCRIPTION
**Pull request by xbtcbridge**
The 'brand' and 'EULA' links in the footer currently return 404. This PR fixes the links.

### CODE REVIEW

File: clarity/contracts/vaults-v2/README.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+liquidate vault, see https://explorer.hiro.so/txid/0x571ffa34bb0171dc275e9f1d1644c72d81e301267227c370426c861d3a161e0d?chain=mainnet
```
Comment (philiphacks) [inline clarity/contracts/vaults-v2/README.txt:1]: REQ: could you remove this file pls?

**Review by philiphacks — CHANGES_REQUESTED** [review]
Thanks for the PR!

Just one small comment

File: clarity/contracts/vaults-v2/README.txt (hunk 1 above)
Comment (xbtcbridge) [reply clarity/contracts/vaults-v2/README.txt]: yes done in [2208a69](https://github.com/arkadiko-dao/arkadiko/pull/591/commits/2208a698685de75bb8715af6bc84e91711aaae9d)

**Review by philiphacks — APPROVED** [review]
Thanks!

**Pull request merged**